lexicons/com/atproto/identity/signPlcOp.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.signPlcOp",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Signs a PLC operation to update some value(s) in the requesting DID's document.",
```

Comment:
does this endpoint also the same safety checks as `sendPlcOp`, or nah?

Reply:
No it doesn't, because it's used to sign an operation for a possibly different service. Though that reminds me, I do intend to put it behind email verification 👍